### DISCUSSION TITLE
How to disable waybar backdrop filter?

### DISCUSSION THREAD

**Discussion by 0xArCHDeViL** [prasanthrangan/hyprdots#1345] [Q&A]
latest HyDE is confusing me fr..

pls tell me how to disable buggy waybar backdrop filter
![240420_16h44m31s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/102850326/6273ecdf-1902-4b65-8431-8a6655099ec0)

**Comment by kRHYME7** [comment]
how did you update?

**Reply by 0xArCHDeViL** [reply]
clean install from minimal arch

**Reply by kRHYME7** [reply]
try ``` Hyde cache rebuild  ``` maybe cache is not fully built

**Reply by dieBakterie** [reply]
With that the black bar is meant right?
